mkshims.ts

```diff
@@ -32,6 +33,7 @@ async function main() {
       const entryPath = path.join(distDir, `${binaryName}.js`);
       const entryScript = [
         `#!/usr/bin/env node`,
+        `process.env.COREPACK_ENABLE_DOWNLOAD_PROMPT??='1'`,
```

Comment:
"??=" may be used only for node 15.14+, but corepack added in 14.19.0+ (https://nodejs.org/docs/latest-v14.x/api/corepack.html) 

If i use node 14.19.3, я get error for command "corepack enable" from last release

Reply:
Corepack requires node.js 18 https://github.com/nodejs/corepack/blob/a3e3e97369dbcdaa2ed6c6ad7c0233ee0a70f221/package.json#L13

If this change slipped into Node.js 14 its probably too late now since it won't receive updates. Its been about a year since the last update (went EOL on 2023-04-30).

https://github.com/nodejs/Release/blob/main/README.md#:~:text=14.x,2023%2D04%2D30